webapp/channels/src/packages/mattermost-redux/src/reducers/entities/general.ts

```diff
@@ -37,6 +38,15 @@ function license(state: ClientLicense = {}, action: MMReduxAction) {
     }
 }
 
+function customProfileAttributes(state: UserPropertyField[] = [], action: MMReduxAction) {
```

Comment:
@sbishel, I missed these in the re-review. Reducer and some related need updating to IDMapped

```suggestion
function customProfileAttributes(state: IDMappedObjects<UserPropertyField> = {}, action: MMReduxAction) {
```

Reply:
No worries...glad you caught it.